cert_manager/certbot.py

```diff
@@ -34,7 +35,15 @@ def request_cert(self, domains):
         for domain in domains:
             command += ["--domain", domain]
         result = subprocess.run(command, stderr=subprocess.PIPE, stdout=subprocess.PIPE)
-        result.check_returncode()
+        try:
+            result.check_returncode()
+        except subprocess.CalledProcessError as e:
+            if self.email_failure_alert:
+                message = "Subject: {subject}\n\n{stderr}".format(
```

Comment:
@ahmedbilal, it will be useful to mention the domains in the message.

Reply:
I think domain is already mentioned by the error generated by certbot